src/invidious/views/community.ecr

```diff
@@ -27,7 +28,7 @@
     </div>
 <% else %>
     <div class="h-box pure-g comments" id="comments">
-        <%= IV::Frontend::Comments.template_youtube(items.not_nil!, locale, thin_mode) %>
+        <%= IV::Frontend::Comments.template_youtube(items.not_nil!, locale, thin_mode, ucid, "community") %>
```

Comment:
It can be a video id, post id or channel id depending on the context (it's channel Id when fetching more community posts while in a channel's community tab, post id is used when fetching for the post page, video id is used when fetching for the watch page)